dockerfiles/template

```diff
@@ -94,6 +94,7 @@ COPY --from={target_name}-build-wrapper /work/cache /work/cache
 COPY cmd/ops_agent_uap_plugin cmd/ops_agent_uap_plugin
 COPY ./builds/ops_agent_plugin.sh .
 RUN ./ops_agent_plugin.sh /work/cache/
+RUN source VERSION && echo $PKG_VERSION  > /work/cache/VERSION
```

Comment:
I think having the same file name can be a bit confusing.
Maybe we can name it something else?

Reply:
Updated to OPS_AGENT_VERSION.